extension/manifest.json

```diff
@@ -32,14 +31,13 @@
 	"background": {
 		"service_worker": "js/background.js"
 	},
-	"permissions": ["tabs", "storage", "identity"],
+	"permissions": ["identity"],
 	"oauth2": {
-		"client_id": "1065384085182-ojd9eluba1ht2lrp6jnfmj295uihlhdc.apps.googleusercontent.com",
+		"client_id": "1093815162744-vrvh40007vs3mvro8715c67sgcacun82.apps.googleusercontent.com",
 		"scopes": [
 			"https://www.googleapis.com/auth/spreadsheets"
 		]
 	},
-	"key": "MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAkz1ANxPg/mvwlv/fn49lKqA85w3z4LSi6StTSZgQXXioU8Ij9FBR6oQCqrU3GH+5Zgjwd55m6ssf+db9Hvinr6pQJUtZXu1QvDET7BcukGka1YgKNLk2HyuhIxYAqa8u2PvUWgrX23DL1iiC6DAQ8y/ARzUbdUs7HA1CwzuWA4L5wsXBC4Cdrostrj4HntcyTVQjCp8kF1zoR5S1G4iLVDGWs/1SPGVJi804XoIBJs43+YzrOrTKQQcwnPI/MY3TO6Fg2ht7oSd4WjVrzVgkxwsuQHuL1zbq7vDSTKWuQgQdU390X9tbWFyzco6O1Lezd1LH5IEt8rCRctiFTXnwkQIDAQAB",
```

Comment:
To investigate: do we need to remove this? If we can keep this here, we could support installing through GitHub as an alternate installation method.

Reply:
I think this is actually best kept not for installing on GitHub, but just for local debugging